<ui version="4.0" >
 <class>MainWindowBase</class>
 <widget class="QMainWindow" name="MainWindowBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" colspan="4" >
     <widget class="QFrame" name="frameMap" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Rubber Band Visibility:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QToolButton" name="mpToolShowRubberBand" >
      <property name="text" >
       <string>Show</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <widget class="QToolButton" name="mpToolHideRubberBand" >
      <property name="text" >
       <string>Hide</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap" >
    <property name="title" >
     <string>Map</string>
    </property>
    <addaction name="mpActionZoomIn" />
    <addaction name="mpActionZoomOut" />
    <addaction name="mpActionPan" />
    <addaction name="mpActionAddLayer" />
   </widget>
   <addaction name="menuMap" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="mpActionZoomIn" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/mActionZoomIn.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
  </action>
  <action name="mpActionZoomOut" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/mActionZoomOut.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="mpActionPan" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/mActionPan.png</iconset>
   </property>
   <property name="text" >
    <string>Pan</string>
   </property>
  </action>
  <action name="mpActionAddLayer" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/mActionAddLayer.png</iconset>
   </property>
   <property name="text" >
    <string>Add Layer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
